packages/core/analytics/core-analytics-server-internal/src/analytics_service.ts

```diff
@@ -29,6 +30,8 @@ export class AnalyticsService {
     });
 
     this.registerBuildInfoAnalyticsContext(core);
+    // Register special `metrics` type
```

Comment:
Nit: I'm not certain that the comment adds enough value here? Should we remove it?